src/web/components/ApiKeyManagement/ApiRolesCell.tsx

```diff
@@ -1,27 +1,27 @@
 import { ApiRoleDTO } from '../../../api/entities/ApiRole';
 import { sortApiRoles } from '../../utils/apiRoles';
+import { Label } from '../Core/Labels/Label';
 import { Tooltip } from '../Core/Tooltip/Tooltip';
 
 import './ApiRolesCell.scss';
-
-function ApiRoleBox({ apiRole }: { apiRole: ApiRoleDTO }) {
-  return <div className='api-role-box'>{apiRole.externalName}</div>;
-}
+import '../Core/Labels/LabelRow.scss';
 
 type ApiRolesProps = {
   apiRoles: ApiRoleDTO[];
   showRoleTooltip?: boolean;
 };
 function ApiRolesCell({ apiRoles, showRoleTooltip = false }: ApiRolesProps) {
+  const sortedRoles = sortApiRoles(apiRoles);
+
   return (
     <div className='api-roles-cell'>
-      <div className='api-role-boxes'>
-        {sortApiRoles(apiRoles).map((role) => (
+      <div className='label-row'>
+        {sortedRoles.map((role) => (
```

Comment:
super nit: i know we are in the `ApiRolesCell` class, but since there are multiple types of roles now, I think it makes more sense to title it `sortedApiRoles`

Reply:
and `role` being named `apiRole`

Reply:
done!